src/Review.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react";
+import reviewsStyling from "./reviewsStyling.module.css";
+import { BiLike, BiDislike } from "react-icons/bi";
+
+function Review(props) {
+  let el = props.param;
+  const [helpful, setHelpful] = useState("Was this review helpful?");
```

Comment:
since these three states are related I recommend using the useReducer react hook.